packages/form-system/stories/01-overview.stories.mdx

```diff
@@ -0,0 +1,101 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { render } from '@lion/core';
+import '@lion/input/lion-input.js';
+import { Validator } from '@lion/validate';
+import './helper-wc/h-output.js';
+
+<Meta title="Form Fundaments/Overview" />
+
+# Form System Overview
+
+This page should be used as a starting point when first using the Form System.
+It provides an overview of its essential building blocks and provides links to detailed explanations
+of most of its core concepts.
+
+## Building Blocks
+
+Our Form System is built from a set of very fundamental building blocks: `form control`s, `field`s and `fieldset`s.
```

Comment:
Nitpick:

Having the final s for the plural form looks funky when it renders.
`form control`s, `field`s and `fieldset`s

What do you think about putting the "s" inside the code block?

```suggestion
Our Form System is built from a set of very fundamental building blocks: `form controls`, `fields` and `fieldsets`.
```

Reply:
The main reason was to highlight the core concepts/defintions in our System. In that sense 'form control' is a concept, 'form controls' not really.
To me this looks better and I think this is the common way to do it. But if you insist on changing it, feel free to do so ;)